### DISCUSSION TITLE
No Apps show up after Fatpack installation

### DISCUSSION THREAD

**Discussion by krassheit** [raspiblitz/raspiblitz#4244] [Q&A]
I installed the latest Fatpack RaspiBlitz on my Raspi 4, the Bitcoin node is 100% syncrhonized for some days now.
Unfortunately, no Apps show up in the Web-UI. There is only this yellow rotating half-circle when I click on the Apps button.
Does anyone know what the problem is?

**Comment by rootzoll** [comment]
Opening an issue on this #4247
